### PR TITLE
Rewrite random string generation

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -13,30 +13,23 @@ import (
...
-const (
-	letterIdxBits = 6                    // 6 bits to represent a letter index
-	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
-	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
-)
 
 var src = rand.NewSource(time.Now().UnixNano())
 
-// RandomString
+// Generate random string with reservoir sampling algorithm https://en.wikipedia.org/wiki/Reservoir_sampling
```
Comment (saheienko) [inline pkg/util/util.go]: Please, update the comment to meet the [conventions](https://blog.golang.org/godoc-documenting-go-code):

```
pkg/util/util.go:19:1:warning: comment on exported function RandomString should be of the form "RandomString ..." (golint)
```

**Comment by gopherstein** [comment]
LGTM

**Comment by gopherstein** [comment]
@stgleb Can you resolve the conflicts here?

**Comment by stgleb** [comment]
@gopherstein Sure, I will

**Pull request merged**